account-management-api/src/main/java/uk/gov/di/accountmanagement/lambda/AuthenticateHandler.java

```diff
@@ -78,26 +68,6 @@ public APIGatewayProxyResponseEvent authenticateRequestHandler(
         try {
             AuthenticateRequest loginRequest =
                     objectMapper.readValue(input.getBody(), AuthenticateRequest.class);
-            var persistentSessionId =
```

Comment:
How comes the account management changes are being reverted? Could add the reason for the revert to the commit and PR description